.circleci/config.base.yml

```diff
@@ -259,6 +257,8 @@ jobs:
 
   amplify_migration_tests_v5:
     <<: *defaults
+    environment:
+      AMPLIFY_PATH: /home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify
```

Comment:
migration tests point to the .npm-global repository, this is where the amplify -g installation will be (v5/v6 only)